src/Admin/Form/RankType.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Forumify\PerscomPlugin\Admin\Form;
+
+use Forumify\PerscomPlugin\Perscom\Entity\Rank;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class RankType extends AbstractType
+{
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver->setDefaults([
+            'data_class' => Rank::class,
+        ]);
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('name', TextType::class, [
+                'required' => false,
+                'empty_data' => '',
+            ]);
```

Comment:
Name field is required. See the `NotBlank` assertion in `Rank.php`.

```suggestion
            ->add('name', TextType::class);
```